sigs/bazel-intellij/CHARTER.md

```diff
@@ -0,0 +1,41 @@
+# Proposed name: Bazel IntelliJ IDE plug-in Special Interest Group
+
+## Objective
+
+The use of the Bazel IntelliJ IDE plug-in is growing and as a result the Google Bazel IntelliJ IDE plug-in team faces increasingly complex questions from customers.
+
+This SIG has the objective of enabling community collaboration and contribution and delivering a good developer experience with the Bazel Intellij IDE Plug-in.  The SIG will:
+
+* cover release updates, new features, issues, and PR requests.
+* Become the “go to” community for Bazel IntelliJ IDE plug-in questions, feedback, and feature requests.
+* Help build skills to successfully use Bazel IntelliJ IDE plug-in.
+* Create and/or shares assets that can be used by users & developers including:
+  * Demos
+  * Monthly updates on new features and product roadmaps
+* Provide a platform for users and developers to regularly share knowledge and learn from each other
+* Connect and foster engagements across various engineering teams.
+
+The SIG will meet once a month.  Members can log issues / topics of interest in bazel-intellij@googlegroups.com.
+
+## Membership
+
+This SIG is open to anybody who uses, develops, or has an interest in sharing or absorbing knowledge on Bazel and the IntelliJ IDE plug-in. Please join bazel-intellij@googlegroups.com and share with interested colleagues.
```

Comment:
I'd add a link to the actual group for simplicity (ie makes it easier to join and to view conversations)
[bazel-intellij@googlegroups.com](https://groups.google.com/g/bazel-intellij)